content/oci/storage.go

```diff
@@ -106,6 +106,22 @@ func (s *Storage) Push(_ context.Context, expected ocispec.Descriptor, content i
 	return nil
 }
 
+// Delete removes the descriptor blob from the storage
+func (s *Storage) Delete(ctx context.Context, target ocispec.Descriptor) error {
+	path, err := blobPath(target.Digest)
+	if err != nil {
+		return fmt.Errorf("%s: %s: %w", target.Digest, target.MediaType, errdef.ErrInvalidDigest)
+	}
+	fullpath := filepath.Join(s.root, path)
```

Comment:
nit: you may want to rename `fullpath` to `target` in terms of consistency.

Reply:
renamed to targetpath